packages/std/src/testing/mock.rs

```diff
@@ -462,11 +462,11 @@ pub type MockQuerierCustomHandlerResult = SystemResult<ContractResult<Binary>>;
 /// MockQuerier holds an immutable table of bank balances
 /// and configurable handlers for Wasm queries and custom queries.
 pub struct MockQuerier<C: DeserializeOwned = Empty> {
-    bank: BankQuerier,
+    pub bank: BankQuerier,
     #[cfg(feature = "staking")]
     staking: StakingQuerier,
     #[cfg(feature = "cosmwasm_1_3")]
-    distribution: DistributionQuerier,
+    pub distribution: DistributionQuerier,
     wasm: WasmQuerier,
     #[cfg(feature = "stargate")]
     ibc: IbcQuerier,
```

Comment:
Shouldn't we make all of them `pub` for consistency and extensibility?

Reply:
The `IbcQuerier` and `WasmQuerier` are currently not public, so I hesitated to do that, but in principle there's nothing preventing us from doing that.

Reply:
We can also do that late. Important is that it is clear what the right(tm) way to do things is from now on and I thought making all public might help.

In this case, let's just double check there are no more straight forward removals we want to do. Then good to merge.

Reply:
Removing the `update_` functions, would look like this on the user side:
```diff
-querier.update_staking(denom, &[], &[]);
+querier.staking = StakingQuerier::new(denom, &[], &[]);
```
So, it might not be quite as obvious that this is something you can do.

Reply:
`querier.update_staking(` can become `querier.staking.update(` which may or may not replace the whole staking querier. A future version of the `StakingQuerier` might contain way more field sucht hat replacing the queries and this relatively simple update can diverge.